.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,29 @@
-#### What is the purpose of this pull request?
-<!--- Describe your changes in detail. -->
-
 #### What problem is this solving?
+
 <!--- What is the motivation and context for this change? -->
 
 #### How should this be manually tested?
 
+[Workspace](url)
+
+#### Checklist/Reminders
+
+- [ ] Updated `README.md`.
+- [ ] If applicable, link this PR to a Clubhouse story.
+- [ ] Whenever possible, create tests for bug fixes.
+- [ ] If applicable, delete workspace after merge.
```

Comment:
Tem que deletar workspace depois do merge?

Reply:
I personally think it's a good practice to use disposable workspaces, in a one-per-PR basis. It allows you to work on multiple things at the same time without losing track of which is which.